segmentstore/server/src/main/java/io/pravega/segmentstore/server/logs/operations/MergeSegmentOperation.java

```diff
@@ -156,12 +181,42 @@ private void write00(MergeSegmentOperation o, RevisionDataOutput target) throws
             target.writeLong(o.streamSegmentOffset);
         }
 
+        private void write01(MergeSegmentOperation o, RevisionDataOutput target) throws IOException {
+            if (o.attributeUpdates == null || o.attributeUpdates.isEmpty()) {
+                return;
+            }
+            target.length(target.getCollectionLength(o.attributeUpdates.size(), ATTRIBUTE_UUID_UPDATE_LENGTH));
+            target.writeCollection(o.attributeUpdates, this::writeAttributeUpdateUUID00);
+        }
+
+        private void writeAttributeUpdateUUID00(RevisionDataOutput target, AttributeUpdate au) throws IOException {
```

Comment:
I know there's no written convention about this, but the suffix of these methods represent the Version & Revision for which they apply. If you began using these in Version 0, Revision 1, then let's set the suffix to 01 please. Same with read.

Reply:
Yeah, I was wondering about this as well, but I will take that non-written rule as if it was written from now on.